src/ivq.tsx

```diff
@@ -76,10 +77,10 @@ export class Ivq extends KalturaPlayer.core.BasePlugin {
       const questionBunchMap = new Map<string, Array<number>>();
       let questionBunch: Array<number> = [];
       qqm.forEach((qq: QuizQuestion) => {
-        questionBunch.push(qq.index);
-        if (qq.startTime === qq.next?.startTime) {
+        if (qq.startTime === qq.prev?.startTime) {
           return;
         }
+        questionBunch.push(qq.index);
```

Comment:
This another bug fix you added?

Reply:
no, fix for markers and their area labels.
Previous implementation - if we have 3 markers in the same time - latest one displayed.
New implementation - if we have 3 markers in the same time - first one displayed.
We need the changes for correct area label on marker.